src/scenic/syntax/scenic.gram

```diff
@@ -854,6 +854,14 @@ scenic_monitor_def_block:
 
 scenic_monitor_statements[list]: a=statement+ { list(itertools.chain.from_iterable(a)) }
 
+# Override
+# --------
+
+scenic_override_stmt:
```

Comment:
For the avoidance of future confusion, let's put a comment explaining that we're restricting `e` to a ` primary` rather than a full expression to avoid possible ambiguities.

Reply:
41f9bd1f0432bacd33fb87521a19300163875fcc

Added a comment. Please let me know if it is not descriptive enough.